src/style.css

```diff
@@ -1,3 +1,84 @@
-body {
+.root {
+  --Height-header: 60px;
+}
+
+a {
+  color: #333;
+}
+
+.all {
+  font-size: 15px;
+  height: var(--Height-header);
   margin: 0;
+  padding: 0;
+  font-weight: 600;
+  font-family: "Roboto", Times, serif;
+  text-transform: capitalize;
+  color: #333;
+}
+
+header {
+  padding: 20px 0;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.logo {
+  display: block;
+  margin-left: 10px;
+}
+
+.nav__list {
+  display: flex;
+  list-style-type: none;
+
+  margin-left: 15px;
+  position: relative;
+  white-space: nowrap;
+
+  align-items: center;
+  color: #333;
+  text-decoration: none;
+  margin-right: 10px;
+}
+
+.is-active {
+  color: blue;
+  border-bottom: 2px solid blue;
+}
+
+@media (max-width: 1024px) {
+  header {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+
+  .nav__list {
+    padding: 0 10px;
+    margin-top: 10px;
+    margin-right: 15px;
+  }
+}
+
+li a {
+  padding: 30px 0;
+  margin-left: 10px;
+  margin-top: 10px;
+  flex-wrap: wrap;
+  text-decoration: none;
+  cursor: pointer;
+  justify-content: flex-start;
+  height: 100%;
+}
+
+@media (max-width: 1200px) {
+  .nav__list {
+    padding: 0 10px;
+    margin-right: 10px;
+  }
+}
+
+:hover {
```

Comment:
we need to target concrete element on which we want to have `:hover` pseudoclass, this won't work like that